Bottom: d161032eb960e0f799cbf18a6a555cff0b9da667
Top:    98729052cb05217bf6462e4de6412c7b107fb219
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-31 17:43:15 +0800

Refresh of vhost_net-simplify

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index a4f7f64..2f7e3bb 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -1025,7 +1025,7 @@ static void vhost_net_flush(struct vhost_net *n)
 		vhost_net_ubuf_put_and_wait(nvq->ubufs);
 		mutex_lock(&vq->mutex);
 		n->tx_flush = false;
-		atomic_set(&n->vqs[VHOST_NET_VQ_TX].ubufs->refcount, 1);
+		atomic_set(&nvq->ubufs->refcount, 1);
 		mutex_unlock(&vq->mutex);
 	}
 }
